test/src/math.c

```diff
@@ -0,0 +1,8 @@
+#include "math.h"
+#include "debug.h"
```

Comment:
Duplicate

Reply:
Is it not standard practice to include the .c file's own header?

Reply:
nvm, just saw that the test/ directory created a bunch of duplicates while I was raging at github for why it wouldn't push my previous commits